src/util/mod.rs

```diff
@@ -387,7 +387,7 @@ impl<L, R> Either<L, R> {
 }
 
 /// `build_info` returns a tuple of Strings that contains build utc time and commit hash.
-fn build_info() -> (String, String) {
+pub fn build_info() -> (String, String) {
```

Comment:
print_tikv_info use info! marco to print the version info, which would introduce some noise like timestamps and line numbers. And we had not initialize the log system yet (at tikv-server.rs:893 but initial_log at tikv-server.rs:910)